- start_x: 12.2
  start_y: 1.25
  start_yaw: 0
  ball_x: 26.45
  ball_y: 10.7
  path:
    - id: 3
      robot: yes
    - id: 4
      robot: no
    - id: 5
      robot: yes
    - id: 15
      robot: no
    - id: 20
      robot: no
    - id: 29
      robot: no
    - id: 41
      robot: no
    - id: 49
      robot: no
    - id: 50
      robot: no
    - id: 42
      robot: no
    - id: 54
      robot: no
  extra_robots: []
  max_duration: 300 #seconds
- start_x: 38.1
  start_y: 43.15
  start_yaw: -1.64535
  ball_x: 36.4
  ball_y: 1.3
  path:
    - id: 146
      robot: yes
    - id: 136
      robot: no
    - id: 119
      robot: no
    - id: 114
      robot: no
    - id: 112
      robot: no
    - id: 100
      robot: no
    - id: 69
      robot: no
    - id: 64
      robot: no
    - id: 55
      robot: no
    - id: 30
      robot: no
    - id: 21
      robot: no
    - id: 17
      robot: yes
    - id: 9
      robot: no
  extra_robots:
    - loc_x: 45.85
      loc_y: 14.95
      yaw: -1.90286
    - loc_x: 30.4
      loc_y: 14.95
      yaw: -1.17601
  max_duration: 300 #seconds
- start_x: 11.8
  start_y: 45.6
  start_yaw: 0
  ball_x: 30.8
  ball_y: 2.85
  path:
    - id: 157
      robot: yes
    - id: 149
      robot: no
    - id: 133
      robot: yes
    - id: 84
      robot: no
    - id: 58
      robot: no
    - id: 50
      robot: no
    - id: 42
      robot: yes
    - id: 33
      robot: no
  extra_robots: []
  max_duration: 300 #seconds
